doc/index.rst

```diff
@@ -1,28 +1,19 @@
 .. _index:
 
-Ubuntu Installation documentation
-#################################
+Ubuntu installation guide
+=========================
 
-A single sentence that says what the product is, succinctly and memorably
-consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-dolore magna aliqua.
+Subiquity is an installer framework for Ubuntu. It provides the Ubuntu Server with a text-based installation user interface and Ubuntu Core first-boot configuration.
 
-A paragraph of one to three short sentences, that describe what the product
-does. Urna cursus eget nunc scelerisque viverra mauris in. Nibh mauris cursus
-mattis molestie a iaculis at vestibulum rhoncus est pellentesque elit. Diam
-phasellus vestibulum lorem sed.
+This documentation set offers guidance for installing Ubuntu Server using Subiquity.
```

Comment:
This is also the home of documentation covering autoinstall for any case, and it is not implausible that Desktop facing documentation may end up here also.

Reply:
I tried to cover that in the note below (seeing that the desktop stuff isn't here yet). 

I'll add info about autoinstall.